creating-pull-requests.md

```diff
@@ -0,0 +1,131 @@
+
+1. Create a new branch with the branch you depend on as a base branch instead of from develop.
+2. When your work is completed, create a PR with that same base branch as the target.
+3. In your description, clearly state that you want the originally blocked PR to be merged before merging this new PR.
+4. If the PR that was blocking you has been merged, change the target branch back to develop.
+
+###### When you want part of your work to be reviewed separately
+
+Sometimes you are working on a part of a feature that you want to have reviewed separately. For example, you may want to check with others that you are on the right path. The workflow will look something like this:
+
+1. Create a separate branch with the feature branch you are currently working on as a base branch.
+2. When the part of the work you want to be reviewed is completed, create a PR back to the feature branch. Make sure that the PR description mentions that this is intended as an intermediate review and that you want it to be merged before merging the full feature.
+
+_After creating the PR, it can happen that it’s more practical to merge the original feature branch_
+
+_to develop first. In this case, it’s still possible to revert back to the steps in When Your work depends on an open PR. This means that after merging the original feature branch, you’ll need to make sure you change the target of your PR back to develop_
+
+### What are your responsibilities in the PR review process?
+
+As the author of the PR, you should maintain the following:
+
+#### Handling feedback
+
+- Read incoming feedback/comments
+- Address feedback/comments by either:
+  - answering the question
+  - making changes to the code
+  - creating new issues for things that should be picked up outside of this PR. When it applies to a specific piece of code, add a TODO comment with a link to that issue in the code.
+- When a comment is resolved, especially when doing so through code changes, add a [✅](https://emojis.wiki/check-mark-button/#:~:text=White%20Heavy%20Check%20Mark%20emoji,emoji%20that%20provides%20positive%20reinforcement.) (:white_check_mark:) reply to let everyone know it has been addressed.
+
+####
+
+Merging into develop
+
+Before merging into develop, merge develop into your branch and resolve any conflicts. If your PR has been open for a long time you might need to repeat this step and make sure your PR still works with the changes from develop.
+
+### General rules for merging a PR
+
+Please read when your PR is ready to be merged below.
+
+- You have at least 2 approvals, one of which is from a senior developer.
+- The CI pipeline build is successful.
+- All feedback from comments is resolved. This can be by resolving all issues raised in the comment, or if necessary by deferring work to a new issue.
+- There are no merge conflicts.
```

Comment:
By default the person that created the PR should also merge it. This is because sometimes extra actions are required after the merge, like updating ticket statuses or notify people.